templates.js

```diff
@@ -382,6 +382,910 @@ module.exports = {
+               width: 100%;
+               height: 100%;
+               border-radius: 1000px;
+               z-index: 1;
+               bottom: 0;
+               left: -70%;
+          }
+           .ppms-popup-header {
+               display: flex;
+               flex-direction: column;
+               align-items: center;
+               z-index: 2;
+          }
+           .ppms-popup-header-sub {
+               font-size: 20px;
+          }
+           .ppms-popup-header-title {
+               text-transform: uppercase;
+               font-weight: bold;
+               color: #086CD4;
+               font-size: 36px;
+               line-height: 44px;
+          }
+           .ppms-popup-form-container {
+               z-index: 2;
+               display: flex;
+               justify-content: center;
+               align-items: center;
+               flex-direction: column;
+               width: 100%;
+          }
+           .ppms-popup-form-container form {
+               display: flex;
+               flex-direction: column;
+               width: 70%;
+          }
+           .ppms-popup-form-info{
+               font-size: 18px;
+               margin-bottom: 30px;
+          }
+           .ppms-popup-form-label {
+               font-size: 14px;
+               margin-bottom: 2px;
+          }
+           .ppms-popup-form-input {
+               height: 35px;
+               border: none;
+               border-radius: 5px;
+               border: 1px solid #919191;
+          }
+           .ppms-popup-form-button {
+               background-color: #086CD4;
+               border: none;
+               color: #fff;
+               font-size: 16px;
+               padding: 10px 25px;
+               text-transform: uppercase;
+               cursor: pointer;
+               box-shadow: 8px 8px 30px -14px rgba(66, 68, 90, 1);
+               align-self: center;
+               margin-top: 20px;
+               border-radius: 0;
+          }
+           .ppms-popup-form-button:hover {
+               background-color: #086CD4;
+          }
+           .ppms-popup-close-button {
+               height: 24px;
+               width: 24px;
+               padding: 4px;
+               z-index: 1000;
+               right: 10px;
+               top: 10px;
+               position: absolute;
+               cursor: pointer;
+               margin: 0;
+               box-sizing: content-box;
+               font-family: sans-serif;
+               font-weight: bold;
+               border-radius: 100%;
+               transition: .3s;
+               color: #000;
+               text-align: center;
+               line-height: 22px;
+               font-size: 24px;
+          }
+           .ppms-popup-close-button:hover {
+               background-color: #ccc;
+               border-color: #ccc;
+               color: #086CD4;
+          }
+          /* These styles affect your popup tracking. Modify with caution. */
```

Comment:
I don't get it. Styles are affecting displaying, not tracking itself - except situation when eg. close button became not visible because of styles derp or something. That note (the same in other templates) might be confusing for clients.